### ISSUE TITLE
Bugs report

### ISSUE THREAD

**Issue by namnm** [namnm/eslint-plugin-sort-keys#3]
@hawtim please move to this thread, thanks

**Comment by hawtim** [comment]
@namnm I found an error when i did not use the rule in overrides, could you help me again.

just like this below.
```js
rules: {
    // override/add rules settings
    // ...
    'sort-keys/sort-keys-fix': 'warn',
  },
  // overrides: [
  //   {
  //     files: [
  //       'src/store/index.js',
  //       'src/views/xSchemas.js',
  //       'src/views/xViews.js',
  //     ],
  //     rules: {

  //     },
  //   },
  // ],
```

![image](https://user-images.githubusercontent.com/11072796/129184457-7093881c-f94a-4f4c-95c7-be976fc5cf52.png)

**Comment by namnm** [comment]
@hawtim Can you check if:
- The plugin is at latest 2.3.4 (I did add some fixes for bugs in case of moving comments)
- Do you know which file in your code it throw this error? I hope we can have something similar to the playground you made so I can examine and fix the bug if any.

**Comment by hawtim** [comment]
yes, it is 2.3.4 in my package-lock.json

```json
"eslint-plugin-sort-keys": {
  "version": "2.3.4",
  "integrity": "sha512-jL9k4dTeHLqcgC7wX2FA7YTCWkpNirP2YMyyxQznuyIN58vSMUXWdBqnH2KSX/3y6ueV0t6gXuElSJeVqYszbg==",
  "dev": true,
  "requires": {
    "natural-compare": "1.4.0"
  }
},
```

I will try to make this error occur in my playground later

**Comment by hawtim** [comment]
I am guessing is it the plugin break this rule somehow.
![image](https://user-images.githubusercontent.com/11072796/129188547-46d8179f-9c0c-4ae4-97ae-ae133d907284.png)

**Comment by hawtim** [comment]
@namnm I found the code cause the error, is the inline comment.

```js
const apiForModule = {
  packToList: {
    receiverList: 'dc/pack/receiver/get', // export page, receiver list
    exportToList: 'pack/export', // export page, export api
    transportOrderCreate: 'create', // to list page, create transport order
    transportOrderDetail: 'fleet/order/search', // toScan page, transport order detail
    transportOrderPrintDetail: 'print/data', // toScan page, transport order print detail for new design
    transportOrderLabel: 'print', // toScan page, transport order label
    completeTransportOrder: 'packed', // toScan page, transport order complete
    reOpenTransportOrder: 'reopen', // toScan page, transport order reopen
  },
};
```

I remove the inline comment and these code will autofix.

**Comment by hawtim** [comment]
I update the eslint-playground

https://github.com/hawtim/eslint-playground

run `npm run sort-keys`

**Comment by namnm** [comment]
@hawtim I published version 2.3.5
